### DISCUSSION TITLE
How to remove 8 players limit for Server

### DISCUSSION THREAD

**Discussion by SenjuZoro** [efroemling/ballistica#271] [Q&A]
Hey peeps,
I would like to ask, how can 8 player limit be removed for server purposes.

**Comment by feroz4u** [comment]
Hey,
Please change the user limit based on your requirement.
_multiteamsession.py
![image](https://user-images.githubusercontent.com/83758326/120325391-3799a100-c305-11eb-9f6a-2decf3d18593.png)
_session.py
![image](https://user-images.githubusercontent.com/83758326/120325450-4718ea00-c305-11eb-8ade-f4f46e137ff0.png)
